applications/add_file_to_db.py

```diff
@@ -134,15 +133,21 @@ def main():
     else:
         pass
 
-    print("Should I insert the following file{} to the {} DB?:\n".format(plural, args.dbToInsertTo))
+    print(
+        "Should I insert the following file{} to the {} DB?:\n".format(
```

Comment:
Who is `I`? Could we not personalize?

Reply:
Sure, it can be `Should the following file {} be inserted to the {} DB?`. I can change this line if you wish, but I would like to merge the open PR first so not mess up my local files.